tools/load-testing/load-gas-loop.ts

```diff
@@ -0,0 +1,141 @@
+import { importAccount, compileSolidity, web3, customRequest, init } from "../init-web3";
+import { TransactionReceipt } from "web3-core";
+import yargs from "yargs";
+import * as rlp from "rlp";
+
+const argv = yargs(process.argv.slice(2))
+  .usage("Usage: $0  [--net stagenet]")
+  .version("1.0.0")
+  .options({
+    net: { type: "string", default: "stagenet" },
+  }).argv;
+
+const contractSource = `
+pragma solidity >=0.4.22;
+
+contract Loop {
```

Comment:
we could reuse this contract bytecode: https://github.com/PureStake/moonbeam/blob/7ceeea985efa01796c6479fe93d9688ed3830d07/tests/tests/constants/testContracts.ts#L133

Reply:
or add this one to the constants

Reply:
I didn't want to have too many dependencies, specially if contract are being used for diiferent purpose.
The one in testcontracts.ts is actually doing some storage instruction and I wanted to limit it to purely CPU bound testing.